packages/react-magma-dom/src/components/NativeSelect/NativeSelect.tsx

```diff
@@ -22,14 +23,41 @@ export interface NativeSelectProps
   /**
    * @internal
    */
-  testId?: string;
   optionLabel?: string;
+  testId?: string;
 }
-const StyledNativeSelect = styled.select<{
+const StyledNativeSelectWrapper = styled.div<{
+  disabled?: boolean;
+  hasError?: boolean;
+  isInverse?: boolean;
   theme: ThemeInterface;
+}>`
+  ${inputWrapperStyles}
+  border:0;
+  svg {
+    color: ${props =>
+      props.isInverse && props.disabled
+        ? transparentize(0.6, props.theme.colors.neutral100)
+        : props.disabled
+        ? transparentize(0.4, props.theme.colors.neutral500)
+        : 'inherit'};
+    margin: 0 0 0 -${props => props.theme.spaceScale.spacing08};
+    pointer-events: none;
+    z-index: 1;
+  }
+`;
+
+const StyledNativeSelect = styled.select<{
   isInverse?: boolean;
+  theme: ThemeInterface;
 }>`
   ${inputBaseStyles};
+  border: 1px solid
+    ${props =>
+      props.isInverse
+        ? transparentize(0.5, props.theme.colors.neutral100)
+        : props.theme.colors.neutral500};
```

Comment:
The border should be red when there's an error, which makes me wonder how the unit tests passed 🤔